WhaleSpotting/react-app/src/components/PageNav.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export default function PageNav() {
```

Comment:
This should also have a return type
```suggestion
export default function PageNav(): JSX.Element {
```
Maybe we can add the rule to the eslint as an error rather than warning? :)

Reply:
Fixed in file. Lint needs exploring